app/services/google/parse_result_service.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Google
+  class ParseResultService
+    TOP_ADS_SECTION_ID = 'tvcap'
+    BOTTOM_ADS_SECTION_ID = 'bottomads'
+    ADS_CLASS = 'uEierd'
+    SEARCH_SECTION_ID = 'search'
+    SEARCH_RESULT_CLASS = 'yuRUbf'
+
+    def initialize(html)
+      @html = html
+    end
+
+    def call
+      @doc = Nokogiri::HTML(html)
+      {
```

Comment:
```suggestion

      {
```

Let's just use an empty line before the "return" statement